test/bultin-addons/core/intl-providers-test.ts

```diff
@@ -343,6 +343,98 @@ for (const asyncFsEnabled of testCaseAsyncFsOptions) {
             ]);
           });
 
+          it('autocompletion bug #1', async () => {
+            expect(
+              (
+                await getResult(
+                  CompletionRequest.method,
+                  connection,
+                  {
+                    app: {
+                      components: {
+                        'test.hbs': `{{t "countries" }}`,
+                      },
+                    },
+                    config: {
+                      'ember-intl.js': `module.exports = function() { return { wrapTranslationsWithNamespace: true } }`,
+                    },
+                    translations: {
+                      'en.yml': `admin:
+  countries: Countries`,
+                    },
+                  },
+                  'app/components/test.hbs',
+                  { line: 0, character: 12 }
+                )
+              ).response
+            ).toEqual([
+              {
+                documentation: 'en : Countries',
+                kind: 12,
+                label: 'admin.countries',
+                textEdit: {
+                  newText: 'admin.countries',
+                  range: {
+                    end: {
+                      character: 5,
+                      line: 0,
+                    },
+                    start: {
+                      character: 5,
+                      line: 0,
+                    },
+                  },
+                },
+              },
+            ]);
+          });
+
+          it('autocompletion bug #2', async () => {
+            expect(
+              (
+                await getResult(
+                  CompletionRequest.method,
+                  connection,
+                  {
+                    app: {
+                      components: {
+                        'test.hbs': `{{t "foo" }}`,
+                      },
+                    },
+                    config: {
+                      'ember-intl.js': `module.exports = function() { return { wrapTranslationsWithNamespace: true } }`,
+                    },
+                    translations: {
+                      'en.yml': `admin:
+  foo: Bar`,
+                    },
+                  },
+                  'app/components/test.hbs',
+                  { line: 0, character: 12 }
```

Comment:
this may fail because pointer is incorrect, it's pointing to line 1 and character 12, we don't have it in `{{t "foo" }}` (pointing to curly, not string)

Reply:
Yep good point 🤦‍♂️ I think I'll close this issue as I don't think this was just a misunderstanding on my part